config_builder.py

```diff
@@ -81,6 +82,7 @@ def build_datadog_conf(self):
         self.set_from_env_mapping('SD_CONSUL_TOKEN', 'consul_token')
         self.set_from_env_mapping('SD_BACKEND_USER', 'sd_backend_username')
         self.set_from_env_mapping('SD_BACKEND_PASSWORD', 'sd_backend_password')
+        self.set_from_env_mapping('SD_JMX_ENABLE', 'sd_jmx_enable')
```

Comment:
This option is [already hardcoded in the JMX variant](https://github.com/DataDog/docker-dd-agent/blob/master/Dockerfile-jmx#L10), and useless in the non-jmx variants. What is your use case for enabling it manually?

Reply:
Ah, right. We build our own container with JMX on alpine. I'll remove this from the PR.